sycl/test/reduction/reduction_nd_s0_dw.cpp

```diff
@@ -0,0 +1,149 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUNx: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+//==----------------reduction_ctor.cpp - SYCL reduction basic test ---------==//
```

Comment:
```suggestion
//==----------------reduction_nd_s0_dw.cpp - SYCL reduction basic test ---------==//
```

Although I think @bader said before that we can drop these licenses from tests, if we want to.

Reply:
That is something new (I see that new sycl/test/abi/* tests don't have it. I dropped licenses.